docs/tasks/announcements.rst

```diff
@@ -1,46 +1,85 @@
-Announcements
-=============
+===============================
+How To Send Email Announcements
```

Comment:
A nitpick, but the precedent I've followed for document titles in other Runbook pages is first word capitalized, all others lowercase, e.g. `How to send email announcements`. It's a nitpick and isn't a required change.